### PR TITLE
[14.0][MIG] apps_download: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT29951

Main changes:
- Remove demo data like was removed in the *github-connector* migration to v14.0. See https://github.com/OCA/interface-github/pull/73#issuecomment-875572690

### CODE REVIEW

File: apps_download/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#~ msgstr "Descripción"
+
+#~ msgid "Hello world"
+#~ msgstr "Hola mundo"
+
+#~ msgid "Title"
+#~ msgstr "Título"
+
+#~ msgid "Module path"
+#~ msgstr "Ruta del módulo"
```
Comment (CarlosRoca13) [inline apps_download/i18n/es.po]:
```suggestion

```

File: apps_download/i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#~ msgstr "Description"
+
+#~ msgid "Hello world"
+#~ msgstr "Hello world"
+
+#~ msgid "Title"
+#~ msgstr "Titre"
+
+#~ msgid "Module path"
+#~ msgstr "Chemin du module"
```
Comment (CarlosRoca13) [inline apps_download/i18n/fr.po]:
```suggestion

```

File: apps_download/i18n/it.po [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+#: model:product.product,uom_name:apps_download.product_product_100b
+#: model:product.template,uom_name:apps_download.product_product_100b_product_template
+msgid "Units"
+msgstr ""
+
+#~ msgid "Description"
+#~ msgstr "Descrizione"
+
+#~ msgid "Title"
+#~ msgstr "Titolo"
```
Comment (CarlosRoca13) [inline apps_download/i18n/it.po]:
```suggestion

```

File: apps_download/i18n/pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+#~ msgstr "Descrição"
+
+#~ msgid "Hello world"
+#~ msgstr "Olá mundo"
+
+#~ msgid "Title"
+#~ msgstr "Título"
+
+#~ msgid "kg"
+#~ msgstr "kg"
```
Comment (CarlosRoca13) [inline apps_download/i18n/pt_BR.po]:
```suggestion

```

File: apps_download/tests/test_app_download.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAppDownload(TransactionCase):
+    """Test for the Add download"""
+
+    def setUp(self):
+        super(TestAppDownload, self).setUp()
```
Comment (CarlosRoca13) [inline apps_download/tests/test_app_download.py]:
```suggestion
        super().setUp()
```

**Review by victoralmau — APPROVED** [review]
Same comments as Carlos.

**Comment by ernestotejeda** [comment]
@victoralmau @CarlosRoca13 changes done

**Review by victoralmau — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**